app/ui/client/components/header/headerRoom.js

```diff
@@ -120,7 +119,7 @@ Template.headerRoom.helpers({
 		const statusText = roomTypes.getUserStatusText(roomData.t, this._id);
 
 		if (s.trim(statusText)) {
-			return emojione.render(s.escapeHTML(statusText));
+			return statusText;
```

Comment:
Is there a plan to add back support for rendering emojis in the status messages?

Reply:
Yes, I just removed it now to make sure the PR is ready for the version cut, but I'll implement the new emoji renderer  once I'm done with some other work I'm busy with.